src/styles/Home.module.css

```diff
@@ -0,0 +1,152 @@
+.container {
```

Comment:
Where are these styles applied? Even with this file deleted I cannot see any changes to the Wallet Connect modal that pops up. Disclaimer - I haven't tried importing any accounts yet.